commit c4230b8db6c9877fc3102e87a40bc24bb9e860c6 (refs/changes/52/5452/3)
Author: Robert Mustacchi <rm@joyent.com>
Date:   2019-02-06T16:03:50+00:00 (8 months ago)
    
    OS-7549 Sync CTF review changes back
    Reviewed by: John Levon <john.levon@joyent.com>
    Reviewed by: Jerry Jelinek <jerry.jelinek@joyent.com>
    Approved by: Jerry Jelinek <jerry.jelinek@joyent.com>

diff --git a/Makefile b/Makefile
index 2fd6b1ca..d4374038 100644
--- a/Makefile
+++ b/Makefile
@@ -74,7 +74,7 @@ CTFBINDIR = \
 	$(ROOT)/projects/illumos/usr/src/tools/proto/*/opt/onbld/bin/i386
 CTFMERGE =	$(CTFBINDIR)/ctfmerge
 CTFCONVERT =	$(CTFBINDIR)/ctfconvert
-ALTCTFCONVERT =	$(CTFBINDIR)/ctfconvert-altexec
+ALTCTFCONVERT =	$(CTFBINDIR)/ctfconvert
 
 SUBDIR_DEFS = \
 	CTFMERGE=$(CTFMERGE) \
diff --git a/pkgsrc/onbld/packlist b/pkgsrc/onbld/packlist
index 12be06c0..eaf4d9b3 100644
--- a/pkgsrc/onbld/packlist
+++ b/pkgsrc/onbld/packlist
@@ -21,7 +21,6 @@
 ./bin/i386/ctfdiff
 ./bin/i386/ctfdump
 ./bin/i386/ctfmerge
-./bin/i386/ctfmerge-altexec
 ./bin/i386/ctfstabs
 ./bin/i386/ctfstrip
 ./bin/i386/cw
diff --git a/src/Makefile.defs b/src/Makefile.defs
index 8ed43227..4b3559e1 100644
--- a/src/Makefile.defs
+++ b/src/Makefile.defs
@@ -42,7 +42,7 @@ GXX =		$(NATIVEDIR)/usr/bin/g++
 CTFBINDIR =	$(ILLUMOS_SRC)/tools/proto/root_i386-nd/opt/onbld/bin/i386
 CTFMERGE =	$(CTFBINDIR)/ctfmerge
 CTFCONVERT =	$(CTFBINDIR)/ctfconvert
-ALTCTFCONV =	$(CTFBINDIR)/ctfconvert-altexec
+ALTCTFCONV =	$(CTFBINDIR)/ctfconvert
 
 SYSINCDIRS =	/usr/include
 SYSLIBDIRS =	/usr/lib /lib
